### PR TITLE
`get_resource_unit` helper method on Metrc::Base

### PR DESCRIPTION
**Pull request by dandehavilland**
~~Deps to be updated when artemis-ag/artemis_api#26 is merged~~

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ else
   gem 'Metrc', git: 'https://github.com/artemis-ag/Metrc.git' # rubocop:disable Bundler/DuplicatedGem
 end
 
-gem 'artemis_api', git: 'https://github.com/artemis-ag/artemis_api'
+gem 'artemis_api', git: 'https://github.com/dandehavilland/artemis_api', branch: 'feature/resource_units'
```
Comment (betacar) [inline Gemfile]: Can you please create a PR to update the gem on it's repo?

Comment (dandehavilland) [reply Gemfile]: Sure, could you give me write access to that repo?

Comment (dandehavilland) [reply Gemfile]: btw, PR already exists here: https://github.com/artemis-ag/artemis_api/pull/26

Comment (betacar) [reply Gemfile]: Done!

**Pull request merged**